🧪 **Casey** — CI Failing ❌

**`lint-and-test (api)` failed:** TypeScript cannot find module `docx`.

```
error TS2307: Cannot find module 'docx' or its corresponding type declarations.
```

The `docx` package needs to be added to `api/package.json` dependencies. Please fix CI before review.

--- a/api/package.json
+++ b/api/package.json
@@ -29,6 +29,7 @@
     "bullmq": "^5.70.1",
     "cookie-parser": "^1.4.7",
     "cors": "^2.8.5",
+    "docx": "^9.6.0",
     "express": "^4.21.0",
     "express-rate-limit": "^8.2.1",
     "handlebars": "^4.7.8",

